internal/config/helpers.go

```diff
@@ -99,5 +97,5 @@ func DumpConfig(logger zerolog.Logger) {
 	configCopy.Azure.ClientSecret = replacement
 	configCopy.GCP.JSON = replacement
 	configCopy.Unleash.Token = replacement
-	logger.Info().Str("binary", path.Base(os.Args[0])).Msgf("Configuration: %+v", configCopy)
+	logger.Info().Msgf("Configuration: %+v", configCopy)
 }
```

Comment:
I moved this field to generic initializer so all pods get it, it is fairly useful.